lib/fastlane/plugin/tunes/actions/tunes_action.rb

```diff
@@ -1,25 +1,56 @@
+require 'shellwords'
 module Fastlane
   module Actions
     class TunesAction < Action
       def self.run(params)
-        sh "afplay '#{params[:file_path]}'"
+        if params[:file_path]
+          play_song(params[:file_path])
+        elsif params[:directory]
+          UI.message("Playing all songs from directory '#{params[:directory]}'")
+          Dir[File.join(params[:directory], "*.mp3")].each do |current|
```

Comment:
We should have a bunch more file extensions here, afplay manpage should list all the supported ones